src/core/org/apache/jmeter/gui/GuiPackage.java

```diff
@@ -71,10 +72,6 @@
     /** Logging. */
     private static final Logger log = LoggingManager.getLoggerForClass();
 
-    private static final String NAMING_POLICY_IMPLEMENTATION = 
-            JMeterUtils.getPropDefault("naming_policy.impl", //$NON-NLS-1$ 
-                    DefaultTreeNodeNamingPolicy.class.getName());
-
     /** Singleton instance. */
     private static GuiPackage guiPack;
```

Comment:
I'm not sure because it's initialized once by GuiPackage.initInstance() called by JMeter#startGui() during the GUI initialization. There seems no concurrency.